labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,11 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "aozejp mysljylc kd kxveddknmc re jsicpdrysirbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcdde kr kd eoya kw aej tysr re ujdr lkgc jv"
+    val output = "yeqour language is impossible to understandthere are twenty six factorial possibilitiesso it is okay if you want to just give up"
+    val dict = input.zip(output).toMap
+    
+    lines.map { line => line.map{ char => dict.apply(char)}}
```

Comment:
For future reference, you can call the apply function of something called foo by just doing foo(args) instead of foo.apply(args). This is, for example, how you can access list indices directly (i.e. if foo is a list then foo(0) is the same as foo.head).